Procfile

```diff
@@ -0,0 +1 @@
+web: node ./bin/www
```

Comment:
Yep, I believe they infer form the procfile what docker image relates to the application you want to tarball and depoly.